tests/conftest.py

```diff
@@ -0,0 +1,187 @@
+'''This file is recognized by pytest for defining specified behaviour
+
+'conftest.py' files are directory-scope files that are shared by all
+sub-directories from where this file is placed. pytest recognises
+'conftest.py' for any unit test executed from within this directory
+tree. This file is used to define fixtures, hooks, plugins, and other
+functionality that can be shared by the unit tests.
+
+This file has been created for the OpenML testing to primarily make use
+of the pytest hooks 'pytest_sessionstart' and 'pytest_sessionfinish',
+which are being used for managing the deletion of local and remote files
+created by the unit tests, run across more than one process.
+
+This design allows one to comment or remove the conftest.py file to
+disable file deletions, without editing any of the test case files.
+
+
+Possible Future: class TestBase from openml/testing.py can be included
+    under this file and there would not be any requirements to import
+    testing.py in each of the unit test modules.
+'''
+
+import os
+import logging
+from typing import List
+
+import openml
+from openml.testing import TestBase
+
+# creating logger for unit test file deletion status
+logger = logging.getLogger("unit_tests")
```

Comment:
This looks a lot like a copy of the logger above. Also, I assume that the logger names should be swapped.